### ISSUE TITLE
Add analysis file for HumanIleumCronsNormalHaberman

### ISSUE THREAD

**Issue by ami-day** [ebi-ait/hca-ebi-wrangler-central#957]
### Project short name:
HumanIleumCronsNormalHaberman

**Project submission (ingest)**
https://contribute.data.humancellatlas.org/submissions/detail?uuid=cddf06c7-5d43-4013-a0b0-271fb611e396&project=504e0cee-1688-40fa-b936-361c4a831f87

**Project in Browser**
https://data.humancellatlas.org/explore/projects/504e0cee-1688-40fa-b936-361c4a831f87/project-matrices

**To do**
**NOT DO DO ANYMORE**
```
- [ ] Create new submission with:
Analysis file
Process

- [ ] Link all the new entities:
Analysis file with previous (or new) analysis protocols
Process with previous cell suspensions

- [ ] Upload new analysis file for GSE134809_RAW.tar into ingest area

- [ ] The existing analysis files might need their matrix_cell_count added

- [ ] The existing analysis_protocol may need the data_normalization_methods and derivation_process fields added.

- New and existing files should use the right Schema:

1. -  Change (or add) analysis protocols to be at least [9.2.0](https://schema.humancellatlas.org/type/protocol/analysis/9.2.0/analysis_protocol)
2.  Change (or add) files with the schema type analysis_file and with version at least [6.3.0](https://schema.humancellatlas.org/type/file/6.3.0/analysis_file)
3. Which references file_core at least [6.2.0](https://schema.humancellatlas.org/core/file/6.2.0/file_core)

- Include the relevant info in the new / existing analysis files:

1.  populate analysis_file.matrix_cell_count
2.  populate analysis_file.file_core.file_source
4.  populate the analysis_protocol with the data_normalization_methods and derivation_process

```
- [ ] Update analysis file with metadata to have file_source as "GEO", and remove "checksum": "GEO" values
- [ ] Put back submission in graph-valid status
- [ ] Export

**Notes**
This is blocked until we have a feature in ingest which enables us to link many cell suspensions (biomaterials) to an analysis file. Currently, linking is manual and not possible to do for many required links.

**Blocked by product dev ticket:**
https://github.com/ebi-ait/dcp-ingest-central/issues/787

**Comment by ami-day** [comment]
It turns out that the analysis files metadata is already present when I download the submission spreadsheet from ingest (and also can be seen in the UI). So something weird has happened because the analysis files are not showing in the HCA Data Browser under this project. I will re-export the project and submit an import form, and see if that resolves this issue.

**Comment by idazucchi** [comment]
the sumbission is in complete status - enrique will kick it to valid

**Comment by ofanobilbao** [comment]
Dependant on #960

**Comment by ESapenaVentura** [comment]
New discoveries have been done

Actions are stated in the body of the ticket

tl;dr: no new files, update existing analysis files, export and import form

**Comment by ami-day** [comment]
Re-exporting in progress

**Comment by ami-day** [comment]
Exported, form submitted.

**Comment by ami-day** [comment]
Needs verification when the December 5th release is live

**Comment by MightyAx** [comment]
Import form submitted.

**Comment by ofanobilbao** [comment]
This is an Update
